packages/gas-station/src/policyUtils.ts

```diff
@@ -0,0 +1,280 @@
+// Policy utilities for Turnkey gas station restrictions
+
+/**
+ * Build a Turnkey policy to restrict what EIP-712 intents an EOA can sign
+ * This protects at the signing layer - EOA cannot create signatures outside policy
+ *
+ * @param config - Policy configuration
+ * @param config.organizationId - Turnkey organization ID
+ * @param config.eoaUserId - Turnkey user ID for the EOA (primary approver)
+ * @param config.additionalApprovers - Optional additional user IDs that must also approve (creates AND condition)
+ * @param config.customConsensus - Optional custom consensus expression (overrides eoaUserId and additionalApprovers)
+ * @param config.restrictions - Signing restrictions
+ * @param config.restrictions.allowedContracts - Whitelist of contract addresses EOA can sign intents for
+ * @param config.restrictions.disallowEthTransfer - Whether to disallow ETH transfers (if true, ethAmount must be 0)
+ * @param config.policyName - Optional policy name
+ * @returns Policy object ready to submit to Turnkey createPolicy API
+ *
+ * @example
```

Comment:
getting a bit verbose, but can we output what the resulting policies would be for each of these as well? would like to be crystal clear on anything policy-related